src/inmanta/data/dataview.py

```diff
@@ -1323,6 +1338,7 @@ def order_and_limit(
             db_order=db_order,
             backward_paging=backward_paging,
             prelude_query_builder=self._prelude_query_builder.order_and_limit(db_order, limit, backward_paging),
+            prelude_query_builder_extra=self._prelude_query_builder_extra,
```

Comment:
Here I think I need to do something (as for the prelude_query_builder) but I don't understand why we need to apply a limit/order on the subqueries, I would have expected doing it only on ht main query was enough